src/app/[locale]/requests/components/Requests.tsx

```diff
@@ -167,6 +171,30 @@ function Requests() {
                 setOpenModerationRequestCount(openMRCount)
                 setClosedModerationRequestCount(closedMRCount)
             })
+        void fetchData('clearingrequests')
+            .then((clearingRequests: EmbeddedClearingRequest) => {
+                let openCRCount = 0
+                let closedCRCount = 0
+                clearingRequests['_embedded']['sw360:clearingRequests']
+                .filter((item: ClearingRequest) => {
+                    if (  item.clearingState === 'NEW' ||
+                          item.clearingState === 'ACCEPTED' ||
+                          item.clearingState === 'IN_QUEUE' ||
+                          item.clearingState === 'IN_PROGRESS' ||
+                          item.clearingState === 'AWAITING_RESPONSE' ||
+                          item.clearingState === 'ON_HOLD' ||
+                          item.clearingState === 'SANITY_CHECK' ||
+                          item.clearingState === 'PENDING_INPUT'){
+                            openCRCount++;
+                    }
+                    else if ( item.clearingState === 'CLOSED' ||
```

Comment:
```suggestion
                    else if (item.clearingState === 'CLOSED' ||
```
Remove redundant space

Reply:
Fixed !